serverless.yml

```diff
@@ -1,36 +1,53 @@
-# For full config options, check the docs:
-#    docs.serverless.com
-
-# the service name is used as a prefix for infrastructure created via
-# CloudFormation. Some infrastructure has limits on name length so don't make
-# the service name too long
 service: projectworks-leave-notify
+useDotenv: true
 frameworkVersion: '3'
+configValidationMode: error
+plugins:
+  - serverless-plugin-scripts
+  - serverless-dotenv-plugin
```

Comment:
We should be able to move off this plugin with `useDotenv` which is built-in to serverless now

Reply:
I read 

> The framework only loads environments variables locally and does not pass them to your function's environment

in https://www.serverless.com/framework/docs/environment-variables and assumed that `useDotEnv` won't let us deprecate (I didn't test it however)

Reply:
That is correct which means it's not a drop-in replacement - so instead you want to define the env variables in the `serverless.yml` config like we do [here](https://github.com/ackama/serverless-aws-template/blob/main/serverless.yml#L36-L38); imo this is better because (among other things) you can check what env variables are used by a lambda without having to hunt around the codebase, and we can enforce an env variable be present or otherwise give it a default value.

Reply:
ah ok I misread it. I have updated it and it seems to work fine in staging